src/ontology/mondo-edit.obo

```diff
@@ -266110,8 +266111,8 @@ xref: OMIM:609734 {source="Orphanet:71526/e", source="MONDO:equivalentTo", sourc
 xref: Orphanet:71526 {source="OMIM:609734", source="MONDO:equivalentTo"}
 xref: SCTID:702949005 {source="MONDO:equivalentTo"}
 xref: UMLS:C1857854 {source="OMIM:609734", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:71526"}
-intersection_of: MONDO:0020075 ! hereditary non-syndromic obesity
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9201 ! POMC
+is_a: MONDO:0011122 {source="https://orcid.org/0000-0001-5208-3432"} ! obesity disorder
```

Comment:
I see many of the `is_a` changes move the term under "obesity disorder", but in this case it does seem to be inherited and am wondering why `consider: MONDO:0019182` (from the term being obsoleted here) is not used?

Reply:
these are inferred to be inherited obesity. I feel like I had a reason why I didn't assert that class but now I forget, to be honest! It may have been I wanted to make sure that they were being inferred?

Reply:
Note, I added a couple more subclassOf axioms, so I think all of the inherited obesity terms are being correctly inferred now